test/JDBC/expected/1_grant_schema.out

```diff
@@ -0,0 +1,2936 @@
+
+select * from s1.t2;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for table t2)~~
+
+insert into s1.t2 values (1);
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for table t2)~~
+
+delete from s1.t2 where a = 1;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for table t2)~~
+
+select * from s1.v2;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for view v2)~~
+
+exec s1.p2;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for procedure p2)~~
+
+select * from s1.f2()
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for function f2)~~
+
+
+-- tsql
+grant select on s1.t1 to u1;
+go
+grant insert on s1.t1 to u1;
+go
+grant delete on s1.t1 to u1;
+go
+grant select on s1.v1 to u1;
+go
+grant execute on s1.p1 to u1;
+go
+grant execute on s1.f1 to u1;
+go
+
+-- tsql user=l1 password=123
+-- Test Group 8 [user u1 has select, insert, execute privilege on these objects]
+select current_user;
+go
+~~START~~
+varchar
+u1
+~~END~~
+
+select * from s1.t1;
+go
+~~START~~
+int
+~~END~~
+
+insert into s1.t1 values (1);
+go
+~~START~~
+varchar
+insert successful
+~~END~~
+
+~~ROW COUNT: 1~~
+
+insert into s1.t1 (a) values (next value for s1.sq1);
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for sequence sq1)~~
+
+delete from s1.t1 where a = 1;
+go
+~~ROW COUNT: 1~~
+
+select * from s1.v1;
+go
+~~START~~
+int
+1
+~~END~~
+
+exec s1.p1;
+go
+~~START~~
+int
+1
+~~END~~
+
+select * from s1.f1()
+go
+~~START~~
+int
+1
+~~END~~
+
+
+-- tsql user=l2 password=123
+select current_user;
+go
+~~START~~
+varchar
+u2
+~~END~~
+
+select * from s1.t1;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for table t1)~~
+
+insert into s1.t1 values (1);
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for table t1)~~
+
+delete from s1.t1 where a = 1;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for table t1)~~
+
+select * from s1.v1;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for view v1)~~
+
+exec s1.p1;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for procedure p1)~~
+
+select * from s1.f1()
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for function f1)~~
+
+
+-- tsql
+grant select on s1.t1 to PUBLIC;
+go
+grant insert, delete on s1.t1 to PUBLIC;
+go
+grant select on s1.v1 to PUBLIC;
+go
+grant execute on s1.p1 to PUBLIC;
+go
+grant execute on s1.f1 to PUBLIC;
+go
+
+-- tsql user=l1 password=123
+-- Test Group 9 [select, insert, execute privilege on these objects to PUBLIC]
+-- [Also, u1 has select, insert and execute privilege on these objects]
+select current_user;
+go
+~~START~~
+varchar
+u1
+~~END~~
+
+select * from s1.t1;
+go
+~~START~~
+int
+~~END~~
+
+insert into s1.t1 values (1);
+go
+~~START~~
+varchar
+insert successful
+~~END~~
+
```

Comment:
should add test where we are now inserting using sequence, and where updates are denied for Group 9 to 17